dali/daliadmin/daadmin.hpp

```diff
@@ -0,0 +1,88 @@
+/*##############################################################################
+
+    HPCC SYSTEMS software Copyright (C) 2021 HPCC Systems®.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+############################################################################## */
+
+#include "dautils.hpp"
+#include "workunit.hpp"
+
+static unsigned daliConnectTimeoutMs = 5000;
+
+extern da_decl void xmlSize(const char *filename, double pc);
```

Comment:
da_decl doesn't look right here.. that's the decl. spec. from the dalibase library.
You _could_ move this functionality into that library, it's already linked by daliadmin and esp services.
But as not needed by most, it would be better to add it to a new library.

Reply:
Will do